simple_history/models.py

```diff
@@ -432,6 +452,10 @@ def create_historical_record(self, instance, history_type, using=None):
         for field in self.fields_included(instance):
             attrs[field.attname] = getattr(instance, field.attname)
 
+        relation_field = getattr(manager.model, "history_relation", None)
+        if relation_field is not None:
+            attrs["history_relation"] = instance
```

Comment:
What happens here when the base object is deleted? I imagine something goes wrong, since you'll have a foreign key to a null object

Reply:
There was a bug which stored the id of the old instance. But now he stores None, which is no problem at all